blockchain/fundamentals/altcoins/altcoins.md

```diff
@@ -21,6 +27,23 @@ tags:
 ---
 ## Content
 
+Bitcoin hasn't been a hit from the beginning. It took a while for people to see for themselves why bitcoins are useful and get used to using them. But there were some people that didn't want to just get used to its limitations: pseudoanonimity, long transaction confirmation time, low transaction throughput, difficulty in obtaining competitive mining hardware and high associated costs, etc.
+
+Bitcoin source code has always been open-source. For someone to start their own blockchain, all they had to do was to copy (*fork*) the source code, find all occurences of `Bitcoin`, `BTC`, and `bitcoin`, replace them with the new name of the coin and run the code.
+
+This method, however, wouldn't change anything to how the system works. One of the first notable altcoins was launched in 2011: `Litecoin`. Litecoin is a fork of Bitcoin and the two blockchains are very similar as a consequence of that.
+
+But similar doesn't mean the same. While both have their coins as the central asset of the blockchain, Litecoin was created to speed up the transactions. Litecoin's blocktime is 2.5 minutes, a quarter of Bitcoin's. To match the time, the maximum supply of litecoins is 4 times the max supply of bitcoins, up to a total of 84 millions.
+
+In terms of consensus algorithm used, Litecoin is secured using proof of work, but this time with a memory-hard hashing algorithm named `Scrypt`, rather than plain *SHA-256*. 
+
+However, Litecoin hasn't managed to solve the anonymity problem Bitcoin had. This justified the creation of new altcoins like `Dash`, `Monero`, or `Zcash`, with a goal of making the users untraceable on the blockchain.
+
+The above mentioned coins and multiple others all have their own merits of improving the decentralized, transaction-focused system that Bitcoin pioneered. But this was nothing new. The true game changer, the blockchain that allowed Turing-complete smart contracts to be deployed and run was `Ethereum`.
+
+Suddenly, blockchain wasn't just about trading coins. New businesses started to follow the trend and release their own token. Ethereum made that process much easier: it has created a standard of tokens that it can recognize and trade on the main network.
+
+`EOS`, `NEO` and `Qtum` are other networks that are meant to be competing with Ethereum on the same smart-contract market.
 
 ---
 ## Practice
```

Comment:
This question could be more about altcoins in the abstract, rather than specific ones